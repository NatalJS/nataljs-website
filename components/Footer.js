import React from 'react';

const Footer = () => (
  <footer id="footer" className="footer">
    <ul className="footer__socials">
      <li>
        <a href="https://www.facebook.com/nataljs/" className="footer__social" title="facebook">
          <img src="/static/images/facebook-logo.svg" className="footer__ico" alt="facebook" />
        </a>
      </li>
      <li>
        <a href="https://github.com/NatalJS" className="footer__social" title="github">
          <img src="/static/images/github-logo.svg" className="footer__ico" alt="github" />
        </a>
      </li>
      <li>
        <a href="https://t.me/natal_js" className="footer__social" title="telegram">
          <img src="/static/images/telegram-logo.svg" className="footer__ico" alt="telegram" />
        </a>
      </li>
    </ul>
    <p className="footer__copy">
      Â© NatalJS.
    </p>
  </footer>
);

export default Footer;
