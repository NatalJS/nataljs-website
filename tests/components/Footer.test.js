
import renderer from 'react-test-renderer';
import Footer from 'components/Footer';

describe('Component Footer', () => {
  let Component;

  beforeAll(() => {
    Component = setup(Footer);
  });

  it('should render component correctly', () => {
    const wrapper = renderer.create(Component());

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
