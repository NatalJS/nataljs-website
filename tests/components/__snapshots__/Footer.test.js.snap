// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Footer should render component correctly 1`] = `
<footer
  className="footer"
  id="footer"
>
  <ul
    className="footer__socials"
  >
    <li>
      <a
        className="footer__social"
        href="https://www.facebook.com/nataljs/"
        title="facebook"
      >
        <img
          alt="facebook"
          className="footer__ico"
          src="/static/images/facebook-logo.svg"
        />
      </a>
    </li>
    <li>
      <a
        className="footer__social"
        href="https://github.com/NatalJS"
        title="github"
      >
        <img
          alt="github"
          className="footer__ico"
          src="/static/images/github-logo.svg"
        />
      </a>
    </li>
  </ul>
  <p
    className="footer__copy"
  >
    Â© NatalJS.
  </p>
</footer>
`;
